### PR TITLE
feat(files): support for upload and download files

### PR DESCRIPTION
**Pull request by enudler**

### CODE REVIEW

**Comment by enudler** [comment]
Breaking changes
1. removing GET: /v1/tests/file/{file_id} and replacing it with GET: v1/files/{file_id}
Predator-runner 1.3.x will be adjusted to use this endpoint (currently used for old processors files)
2. OpenAPI/Docs will be in another branch

File: docs/openapi3.yaml [hunk 1]
```diff
@@ -1350,6 +1354,72 @@ paths:
...
+      description: Retrieve a specific file.
+      parameters:
+        - in: path
+          name: file_id
+          description: The id of file to download
+          required: true
+          schema:
+            type: string
+      responses:
+        '201':
```
Comment (NivLipetz) [inline docs/openapi3.yaml:1416]:
```suggestion
        '200':
```

File: tests/integration-tests/files/files-test.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        });
+
+        it('big csv upload should result in 413', async () => {
+            const response = await requestCreator.uploadFile('csv', BIG_CSV_PATH);
+            should(response.statusCode).eql(413);
+        });
+    });
+
+    describe('download file', () => {
+        it('Download existing file', async () => {
```
Comment (NivLipetz) [inline tests/integration-tests/files/files-test.js:42]:
```suggestion
        it('Download not exiting file, async () => {
```

File: tests/integration-tests/files/files-test.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            const response = await requestCreator.downloadFile('10bc7982-d57b-4917-a952-7a8d8d75bf92');
+            should(response.statusCode).eql(404);
+        });
+
+        it('Download not exiting file id', async () => {
```
Comment (NivLipetz) [inline tests/integration-tests/files/files-test.js:47]:
```suggestion
        it('Download existing file', async () => {
```

File: docs/openapi3.yaml [hunk 4]
```diff
@@ -1350,6 +1354,72 @@ paths:
...
+        - Files
+      summary: Download file
+      description: Retrieve a specific file.
+      parameters:
+        - in: path
+          name: file_id
+          description: The id of file to download
+          required: true
+          schema:
+            type: string
```
Comment (NivLipetz) [inline docs/openapi3.yaml:1414]:
```suggestion
            type: string
            format: uuid
```

**Pull request merged**